korma/src/commonMain/kotlin/com/soywiz/korma/geom/Rectangle.kt

```diff
@@ -4,10 +4,10 @@ import com.soywiz.korma.internal.*
 import com.soywiz.korma.interpolation.*
 
 interface IRectangle {
```

Comment:
Maybe you can extend IPoint now?

Reply:
Seems you are talking about korlibs/korma#44. Yep, I'm planning to do it in another PR, after we deal with this one.